docs/content/FAQ/_index.md

```diff
@@ -36,19 +40,35 @@ If you want to see what has changed between versions, refer to the [releases pag
 
 The Infection Monkey agent shuts off either when it can't find new victims or it has exceeded the quota of victims as defined in the configuration.
 
+## Where does Infection Monkey store runtime artifacts?
+
+### Monkey Island server
+
+The Island server will need to store some files on the system on which it is running, such as the Island logs and any custom post-breach action files.
+
+For this, it uses a "data directory" which can be configured in the `data_dir` field in the `server_config.json` file. The default data directory on Linux is `$HOME/.monkey_island`, and on Windows is `%AppData%\monkey_island`.
```

Comment:
How is this a frequently asked quesiton? In which case, I as a user, would care about where island stores some files?

Reply:
I think it makes sense to add it. People may want to know exactly how and where the Island will modify the system(s). It's better having it with no one looking for it, than not having it with many people looking for it and not finding it. Same for the comment below.

I'm adding it back for now, but if you still strongly feel we should leave it out, let's talk about it.

Reply:
I think this could be useful when we get support questions. Some answers may require pointing users at the data directory. I'd rather link them to the docs for this information than re-explain it every time.

Reply:
> Some answers may require pointing users at the data directory. I'd rather link them to the docs for this information than re-explain it every time.
FAQ should be brief and cover common scenarios/questions. Data dir is a concern of technical implementation. What is data-dir can be explained elsewhere, with other implementation-related topics, **in references perhaps**? If we'll get questions like "where is data dir?", we can add it later. For now, it would make more sense to point to data dir for questions like "how do I reset password", without explaining what data dir is, because the password can be reset without that knowledge.